src/NHibernate/AdoNet/Util/BasicFormatter.cs

```diff
@@ -441,6 +442,11 @@ private void Newline()
 				}
 				beginLine = true;
 			}
+
+			public void Dispose()
+			{
+				tokens.Dispose();
```

Comment:
This one is a bit nit-picking: concrete implementation of this `Dispose` is currently no-op.